Automattic/wp-calypso
# Stats: Updating Subscribers summary to use real data

Related to #76784

## Proposed Changes

* use rest endpoint to display real data for Subscribers `All time` stats

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug`
* Add the query string `?flags=stats/subscribers-section` to enable the subscriber counts page
* Go to the `Subscribers` page
* Verify that the data at the top of the page in `All time stats` changes when you navigate to another blog

## Pre-merge Checklist
